tst/DataChannelFunctionalityTest.cpp

```diff
@@ -92,6 +92,78 @@ TEST_F(DataChannelFunctionalityTest, createDataChannel_Disconnected)
     ASSERT_EQ(1u, remoteOpen.channels.at("Answer PeerConnection"));
 }
 
+TEST_F(DataChannelFunctionalityTest, dataChannelSendRecvMessageAfterDtlsCompleted)
+{
+    RtcConfiguration configuration;
+    PRtcPeerConnection offerPc = NULL, answerPc = NULL;
+    PRtcDataChannel pOfferDataChannel = nullptr, pAnswerDataChannel = nullptr;
+    SIZE_T pOfferRemoteDataChannel = 0, pAnswerRemoteDataChannel = 0;
+    SIZE_T datachannelLocalOpenCount = 0, msgCount = 0;
+    BOOL dtlsCompleted = FALSE;
+
+    MEMSET(&configuration, 0x00, SIZEOF(RtcConfiguration));
+
+    auto onDataChannel = [](UINT64 customData, PRtcDataChannel pRtcDataChannel) {
+        ATOMIC_STORE((PSIZE_T) customData, (SIZE_T) pRtcDataChannel);
+    };
+
+    auto dataChannelOnOpenCallback = [](UINT64 customData, PRtcDataChannel pDataChannel) {
+        UNUSED_PARAM(pDataChannel);
+        ATOMIC_INCREMENT((PSIZE_T) customData);
+    };
+
+    auto dataChannelOnMessageCallback = [](UINT64 customData, PRtcDataChannel pDataChannel, BOOL isBinary, PBYTE pMsg, UINT32 pMsgLen) {
+        UNUSED_PARAM(pDataChannel);
+        UNUSED_PARAM(isBinary);
+        if (STRNCMP((PCHAR) pMsg, TEST_DATA_CHANNEL_MESSAGE, pMsgLen) == 0) {
+            ATOMIC_INCREMENT((PSIZE_T) customData);
+        }
+    };
+
+    EXPECT_EQ(createPeerConnection(&configuration, &offerPc), STATUS_SUCCESS);
```

Comment:
Minor: These macros expect the first param to be the expected value and the second the evaluated one

Reply:
this is already prevalent in the test, would be better to fix them all at once in a separate pr.

Reply:
yeah of course